portfolio/src/main/webapp/script.js

```diff
@@ -28,21 +28,17 @@ function addRandomFact() {
 }
 
 
-function getFromServlet(){
-    fetch('/data').then(response => response.text()).then((comment) => { 
-
-    const commentContainer = document.getElementById('comment-container');
-    console.assert(commentContainer, "You have no element with ID 'comment-container'"); 
-    
-    
-        const list = document.getElementById('comment-container');
-        comment = comment.replace("[", "");
-        comment = comment.replace("]","");
-        var commentArray = comment.split(',');
-        for(i =0;  i < commentArray.length; i++){
-            list.appendChild(createListElement(commentArray[i]));
+function getFromServlet(value){
```

Comment:
"populate" may be more appropriate than "get", since you aren't returning anything from this method

Reply:
I think I might just change the name entirely. Do you think "generateComments" would be an appropriate name?

Reply:
Yea that works